src/java.base/share/classes/java/lang/System.java

```diff
@@ -736,7 +736,12 @@ public static native void arraycopy(Object src,  int  srcPos,
      *     <td>Java Runtime Environment specification maintenance version, defined only
      *         when the specification implemented by this runtime has undergone a
      *         <a href="https://jcp.org/en/procedures/jcp2#3.6.4">maintenance
-     *         release</a> <em>(optional)</em></td></tr>
+     *         release</a> <em>(optional)</em>.
+     *         After a first maintenance release, this property will
```

Comment:
Separate independent clauses with a semicolon: `After a first maintenance release this property will have the value {@code "1"}; after a second maintenance release it will have the value {@code "2"}, and so on.`

Reply:
PR and CSR updated.